### ISSUE TITLE
O> Outline Scopes of the Governance Working Group into a document

### ISSUE THREAD

**Issue by patrick727** [rchain/bounties#133]
1. Take Kenny summary "Identification and selection of directors, officers, and other important roles in the Cooperative, subject to Board approval. Oversee selection of Executive Committee general membership." clarify each point. 

2. Propose/discuss ideas that members consider related to "Governance"

3. Outline and Draft "Governance Committee" overview

**Comment by Keaycee** [comment]
Hi Patrick
I could not find any summary on the above topic written by Kenny Rowe. Please if you do have a link to the above topic, it will be very needed.

**Comment by Keaycee** [comment]
Hello,

Base on the issue of Governance committee, i have done a comprehensive work on it. I want the members of the cooperative to make a review on it. Below is the document link.

https://docs.google.com/document/d/1U3elyL4y-H8Z6uBU61tiO0sRTWu7DeRmPjfdYgpUOKY/edit?usp=sharing

**Comment by patrick727** [comment]
woah great work @Keaycee 

can you join the governance meetings thursdays? 

I just lightly reviewed it and will go more in depth as we move forward.

Cheers

**Comment by Jake-Gillberg** [comment]
@Keaycee Added a few comments to the doc.

**Comment by Keaycee** [comment]
Thank you, for reviewing the issue. I will make a correction base on the your suggestions.

**Comment by Keaycee** [comment]
Yes i can @patrick727

**Comment by Keaycee** [comment]
Corrections and suggestions has been has been done. However, there may still be need for further review.

https://docs.google.com/document/d/1U3elyL4y-H8Z6uBU61tiO0sRTWu7DeRmPjfdYgpUOKY/edit?usp=sharing

**Comment by lapin7** [comment]
I've set the budget of #133 arbitrarily to $0.00 because of the discussion **volunteer and rewards**
Please discuss amongst others with @dckc

**Comment by dckc** [comment]
@patrick727 point 2 in the description muddies the waters considerably. The RChain Nomination and Governance Committee has no mandate beyond "Identification and selection of directors, officers, and other important roles in the Cooperative, subject to Board approval. Oversee selection of Executive Committee general membership."

The use of "Governance Committee" in the issue title made me wonder if you were referring to the same committee introduced in the bylaws, created by the board, and discussed at the annual meeting, but your point 1 clarifies that you are.

As I said in my [Jan 27 comment][Jan27], broader governance work is interesting, but it's separate from the work of the RChain Nomination and Governance Committee.

Anyone who remains unclear on this, please discuss with officers such as @kennyrowe and/or board members such as @drbloom .

[Jan27]: https://github.com/rchain/Members/issues/229#issuecomment-360992218

**Comment by jimscarver** [comment]
Indeed there has been some confusion. My takeaway from the annual meeting was the governance committee responsibility went far beyond the specific responsibilities listed in the bylaws including block chain and coop governance with Vlad as a guide.

I took on the task of initiating activities to forward our mandate as I understood it which may have led to confusion. My apologies for my misunderstanding.

In the [RGov log](https://docs.google.com/document/d/1PwLD4eQV-w42iY2oKuBHV-PItQ40dv_Qq_6L1IDLK0k/edit) I suggested changing the name to "governance working group" when the official "RChain Nomination and Governance Committee" would be a subset of working group members.

I suggest @Keaycee s doc be proposed as the scope of this "RChain Governance Working Group". This is important work that needs to be done.

**Comment by dckc** [comment]
Thanks, @jimscarver . I'm inclined to revise the title and description of this issue to refer to a "governance working group". I'll stand for a bit by to see if @patrick727 or you or anybody else would rather do it yourself.

**Comment by dckc** [comment]
Which part of the scope document is original work?

Prompted by [@drbloom 's comment][1] I took an arbitrary sentence from another section and looked it up in google and found that the entire section on "The Meaning of Decentralization" is copied without attribution from https://medium.com/@VitalikButerin/the-meaning-of-decentralization-a0c92b76a274


[1]: https://github.com/rchain/Members/issues/306#issuecomment-364898520

**Comment by dckc** [comment]
I'm very much conflicted... I think **wholesale copying without attribution is behavior deserving immediate and permanent revocation of contributor privileges**. And in #319, @lapin7 made me an admin. So now if I don't remove the author's contributor status in this repository, I'm complicit.

I'll stand by for a day or two for suitable explanation.

**Comment by pythonical** [comment]
Is there a blurb about intellectual theft on the member sign up document? If not there should be an ‘acceptable use’ policy.

**Comment by lapin7** [comment]
@dckc I suggest that @Keaycee doesn't get any reward for his work done in February. Everywhere he appears in the reward sheet, I will downvote him. Others can do the same. 

Copying like this and claiming that you wrote it, is an awful behavior.
**@Keaycee please explain this action!!!**

**Comment by Keaycee** [comment]
My work is base on research and contribution from experienced experts online. However the work is still on progress and still under review. After proper corrections and review, **conclusion** and **reference** will be added. 
I made research and got this put together, but if my work (Document) is not worth anything to RChain Cooperative then anybody can act as they see fit.

**Comment by dckc** [comment]
Nothing in your comment answers my question: Which part of the scope document is original work? You have not indicated that a single sentence or paragraph is original work.

**Comment by Keaycee** [comment]
@jimscarver  @patrick727 @Jake-Gillberg @lapin7 @pythonical 
**Summary
Conclusion
Recommendation
Reference**

The above has been added to the document. Suggested correction were made. 
This work is completed, although there maybe need for further review and suggestions.

**Comment by dckc** [comment]
The blatant plagiarism remains.

Comments (including specific requests that quoted material be marked as such) are closed without being addressed. Repeatedly: 1:16 PM Jan 6, 11:49 AM Feb 14, 11:50 AM Feb 14, 8:32 AM Today.

@patrick727 @jimscarver do you see any reason why I shouldn't revoke @Keaycee 's privileges?

**Comment by pythonical** [comment]
i really hate seeing this having to take place but it does make me wonder is there any process for 'reviews' of such cases and is/will there be any recourse for someone to plead their case after the fact?

**Comment by Ojimadu** [comment]
Revocation of member's rights should not be based on a decision by a single member. There needs to be a review of cases like this and not based off an arbitrary decision.

**Comment by Keaycee** [comment]
Well, I don't where i have gone wrong, was it because i resolved a comment on my work after i have corrected and taken note of it OR was it because i sourced for extra information online to support the document?. The document is open and anyone can make a suggestion by editing after which i will accept the suggestions and corrections made  just like @jimscarver always do. He reviews the document, make some corrections and suggestion by editing it. 
Why is it a cooperative? Because it involves mutual assistance in working towards a common goal. @keaycee alone can't do all the work and corrections, that is why we are cooperative in other to assist in getting a work done properly.  Everyone reviewing this work has access to edit and make corrections to the document. 

@lapin7 If Mr. Keaycee works on issues A B C and D, and his work on issue B seem not to be good enough, Do you think it's proper or right to deny him a reward on other issues A C and D?
I never claimed to have written all by my self and if i did please quote me where and when i did mention that. My work is based on research and it's still open for any further research.

There are more grieve mistake and actions of a coop member that could lead to a revoke of privilege and i do not see resolving a comment after been corrected and noted as one of it. Except it's a personal act and not official. 

This cooperative is decentralized, we can only work better to get the best result if we assist and support one another than taking ones mistake or error personally. It will only lead to discouragement of coop members. 

@jimscarver please review the document for further correction and suggestions.

Thank you.

**Comment by Keaycee** [comment]
@patrick727 @Jake-Gillberg @lapin7 @jimscarver @dckc @pythonical  I'm very sorry if about everything and I accept my mistakes and wrongs. I really do not feel nice about the way this is going. Issue 133 was the first work I was assigned to by @patrick727 when I became a member of the coop. I'm new to collaborative work and this was the first issue I worked on. I tried my best to make corrections and also reference on how my work came about, but it seem the plagiarism is very obvious.
I suggest this document be removed since it's not yet a working document and has not yet been adopted. 
I also suggest that my reward for the work be removed and I also should be unassigned for the work.

**Comment by pythonical** [comment]
@Ojimadu others have looked at it and commented along the thread so it isn’t just down to one. I am a bit disturbed by these last comments by keaycee. Something odd.

**Comment by Ojimadu** [comment]
@pythonical I was not referring to you or anyone in particular.

**Comment by dckc** [comment]
I looked more closely into the history of contributions by @Keaycee and I found a good conversation on telegram in Africa (IOU the #).

I'm light of that contribution, I am inclined to look at this as a misunderstanding. I accept your apology.

I see @lapin7 is making more comments to clarify norms for quoting work by others. That's good, but it could be a slow process, and this really should be addressed quickly.

Would a co-author please step forward and request edit access?

Or should we mark this document obsolete and begin again later?

**Comment by dckc** [comment]
I got some input from @jimscarver ; he encouraged me to not just request changes but demonstrate what I have in mind. I replaced wholesale copy and paste sections by placeholders: _pending appropriate excerpt / summary._

I'm not available to do the excerpts / summaries just now. I hope another collaborator comes along for that.

The remaining text, starting with the **GOVERNANCE working group MODEL** also seems to be copied and pasted from other sources, but (a) it seems to be a somewhat novel synthesis, and (b) those sources are coop governance materials with a reasonably clear heritage of liberal copying and pasting.

I also added a byline with some status info an a link to this issue for context.
